src/packages/version-check/tests/semver.test.ts

```diff
@@ -0,0 +1,585 @@
+// @ts-nocheck
+process.env.VERSION_CHECK_CONFIG_NAME = "testConfig";
+import { isValidSemver, semverUpgradeType } from "../src/semver";
+import assert from "assert";
+
+describe("semverUpgradeType", () => {
+  const testVersion = "0.0.0";
+  const version = "1.2.3";
+  const alphaVersion = "1.2.3-alpha";
+  const betaVersion = "1.2.3-beta";
+  const invalidVersion = "notasemver";
+
+  let message;
+
+  beforeEach(() => {
+    message = "";
+  });
+
+  describe("nulls", () => {
+    it("handles null currentVersion", () => {
+      assert(
```

Comment:
These asserts are wrong, the signature for `assert` is `assert(value: unknown, message?: string | Error)`; where value is checked to be truthy and `message` is shown if the test false. In this case message is just `true`.

Can you update these all of these tests to use the more precise assert methods?

Reply:
:+1: